### ISSUE TITLE
 could you please tell me how to run the test set?

### ISSUE THREAD

**Issue by 1553545365** [Song-Jingyu/PointPainting#3]
Could you please tell me how to run the test set?

I run : "python test.py --cfg_file cfgs/kitti_models/pointpillar_painted.yaml --batch_size 1 --ckpt /home/wenhao/桌面/PointPainting-main/detector/output/kitti_models/pointpillar_painted/default/ckpt/checkpoint_epoch_80.pth".

But the result of "pointpillar+DeepLab V3+" seemed  have no different with  original "pointpillar"

**Comment by Song-Jingyu** [comment]
Hi,

There should be at least some difference with original PointPillar. When you use the PointPillar, you may verify if you have used the original ckpt trained by PointPillar config.

**Comment by 1553545365** [comment]
> Hi,
> 
> There should be at least some difference with original PointPillar. When you use the PointPillar, you may verify if you have used the original ckpt trained by PointPillar config.


wowowow!  Thank you for your reply. Do you mean my following command correct?
( "python test.py --cfg_file cfgs/kitti_models/pointpillar_painted.yaml --batch_size 1 --ckpt /home/wenhao/桌面/PointPainting-main/detector/output/kitti_models/pointpillar_painted/default/ckpt/checkpoint_epoch_80.pth")

**Comment by Song-Jingyu** [comment]
I think it should be right! Actually I did not run test.py. The test.py is actually doing evaluation, which is similar to the output of training. I believe your command is correct.

**Comment by 1553545365** [comment]
> I think it should be right! Actually I did not run test.py. The test.py is actually doing evaluation, which is similar to the output of training. I believe your command is correct.

In fact, I've got the results of the test set, but I'm not sure if the results have been Pointpainted?
This is my test datapath:
detector
├── data
│ ├── kitti
│ │ │── ImageSets
│ │ │── training
│ │ │ ├── calib
│ │ │ ├── image_2
│ │ │ ├── image_3
│ │ │ ├── label_2
│ │ │ ├── velodyne
│ │ │ ├── planes
│ │ │ ├── painted_lidar
│ │ │── testing
│ │ │ ├── calib
│ │ │ ├── image_2
│ │ │ ├── velodyne

Do I need to run painting.py for the test set, before I running test.py?
Thank you soooo mach.

**Comment by Song-Jingyu** [comment]
Hi,

I am quite busy these days. I think you might misunderstand the test.py function, it still aims to use the validation set to test, not test set. I did not check how to test capability on test set. I think you may check OpenPCDet for more details.

Best,
Jingyu
